fml/thread_unittests.cc

```diff
@@ -45,15 +79,27 @@ TEST(Thread, ThreadNameCreatedWithConfig) {
   bool done = false;
   thread.GetTaskRunner()->PostTask([&done, &name]() {
     done = true;
-    char thread_name[8];
+    char thread_name[16];
     pthread_t current_thread = pthread_self();
-    pthread_getname_np(current_thread, thread_name, 8);
+    pthread_getname_np(current_thread, thread_name, 16);
     ASSERT_EQ(thread_name, name);
   });
   thread.Join();
   ASSERT_TRUE(done);
 }
 
+static int clamp_priority(int priority, int policy) {
+  int min = sched_get_priority_min(policy);
+  int max = sched_get_priority_max(policy);
+  if (priority < min) {
```

Comment:
Can use `std::clamp(priority, min, max)` here

Reply:
I guess my head is stuck in c++ 14 :)